libraries/cd/cd-footer/cd-ocha.css

```diff
@@ -1,102 +1,105 @@
 /**
- * Common Design: Sites Menu ('OCHA Services')
+ * Common Design: OCHA Services
  */
-.cd-ocha h2 {
-  color: var(--cd-white);
+.cd-ocha-services {
+  @media screen and (min-width: 768px) {
+    padding-block-start: 2rem;
+    position: relative;
+
+    /* Add a horizontal separator once layout expands */
+    &::before {
+      content: '';
+      display: block;
+      width: 100%;
+      height: 1px;
+      position: absolute;
+      top: 0;
+      left: 0;
+      background: var(--cd-white);
+      opacity: 60%;
+    }
+  }
 }
 
-.cd-ocha__btn {
-  position: relative;
-  display: flex;
-  align-items: center;
-  height: var(--cd-global-header-height);
-  padding: 0 var(--cd-outline-size);
-  transition: background 0.3s ease;
+.cd-ocha-services h2 {
+  margin-block-end: 0;
   color: var(--cd-white);
-  background: transparent;
-  font-size: var(--cd-font-size--tiny);
+  font-size: var(--cd-font-size--medium);
 }
 
-.cd-ocha__btn:focus-visible {
-  outline-offset: calc(0px - var(--cd-outline-size));
-}
-
-.cd-ocha__btn .cd-icon--arrow-down {
-  margin: auto; /* Center vertically in macOS 10 */
-}
-
-.cd-ocha__logo {
-  display: inline-block;
-  flex-shrink: 0;
-  width: 21px;
-  height: 21px;
-  fill: var(--cd-white);
-}
-
-.cd-ocha-dropdown {
+.cd-ocha-services__content {
   width: 100%;
-  padding: 22px 0;
+  padding: 1.25rem 0;
 }
 
-.cd-ocha-dropdown__inner {
+.cd-ocha-services__inner {
   display: flex;
   flex-flow: column nowrap;
   gap: 0 12px;
   max-width: var(--cd-max-width);
-  padding: 0 var(--cd-container-padding);
+  padding: 0;
 }
 
-.cd-ocha-dropdown__section {
+.cd-ocha-services__section {
   position: relative; /* For the see-all link */
   margin-bottom: 1.5rem;
 }
 
-.cd-ocha-dropdown__heading {
+.cd-ocha-services__heading {
   margin: 0 0 10px;
   padding-bottom: 2px;
   color: var(--brand-grey);
 }
 
-.cd-ocha-dropdown__list {
+.cd-ocha-services__list {
   margin: 0;
   padding: 0;
 }
 
+/**
+ * backwards-compat: the .cd-ocha-dropdown class is legacy, but is used by lots
+ * of sub-themes which customized their OCHA Services links. If these classnames
+ * get removed then the styling of the first column will look strange.
+ */
+.cd-ocha-services__link,
```

Comment:
Classnames have been revamped to remove the reference to a dropdown, with the exception of these three rules accompanied by the comment.